src/Metadata/Property/Factory/ExtractorPropertyMetadataFactory.php

```diff
@@ -95,10 +95,27 @@ private function update(ApiProperty $propertyMetadata, array $metadata): ApiProp
             'description' => 'get',
             'readable' => 'is',
             'writable' => 'is',
-            'writableLink' => 'is',
```

Comment:
Can you don't move existing keys (to ease merge commits) please?

Reply:
@dunglas OK, I've fixed and force-pushed as 624105c. Thanks.